packages/s/snapd/files/wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+YELLOW='\033[0;33m'
+NC='\033[0m'
+URL=https://help.getsol.us/docs/user/software/third-party/snap
+SNAP=/usr/lib64/snapd/snap
+CONFIG="/var/lib/snapd/solus"
+CONFINEMENT="$("${SNAP}" debug confinement 2>/dev/null)"
+
+if [ -e "${CONFIG}" ]
+then
+  # shellcheck disable=SC1090
+  . "${CONFIG}"
+fi
+
+if [ "$#" -ge 1 ] && [ "$1" = "hide-confinement-warning" ]
```

Comment:
If this was a double-square-bracket `[[ ]]` bash conditional, you could use `==` ...?

Reply:
Yep!